src/verifier/snp/milan_ark_ask.pem

```diff
@@ -0,0 +1,74 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
if ASK is first in this file, then maybe rename it to `milan_ask_ark.pem` ?

Reply:
Turns out the ARK is first after all.

Reply:
Actually, the ASK is first @fitzthum (at least as of Feb 7, 2023).